x-pack/plugins/canvas/public/style/index.scss

```diff
@@ -53,7 +53,8 @@
 @import '../components/workpad_export/workpad_export';
 @import '../components/workpad_loader/workpad_loader';
 @import '../components/workpad_loader/workpad_dropzone/workpad_dropzone';
-@import '../components/workpad_page/workpad_page';
+@import '../components/workpad_interactive_page/component';
```

Comment:
nit: `component.scss` is not a convention we've used before. `'../components/workpad_interactive_page/workpad_interactive_page` would be more in line... even better would be to keep both sets of styles in `'../components/workpad_page/workpad_page`

Reply:
Done, will push in a bit